### DISCUSSION TITLE
Set up Rowy Run to use Derivative fields

### DISCUSSION THREAD

**Discussion by organicnz** [AntlerVC/firetable#801] [Support / Q&A] [answered]
**Describe the bug**
Whenever I try to add a derivative, a frame appears with the message "Set up Rowy Run to use Derivative fields" but it's already set up. 

**To Reproduce**
1. Go to 'dictionary_api' column.
2. See error 'Set up Rowy Run to use Derivative fields'.

**Expected behavior**
'Set up Rowy Run' should be run once as it was before.

**Screenshots**
https://www.loom.com/i/99891efe7ce749bc982b60eaefbe5e0d

**Rowy Run version**
Rowy v3.0.0

**Desktop (please complete the following information):**
 - OS: macOS Monterey
 - Browser Chrome
 - Version 105

**Smartphone (please complete the following information):**

**Additional context**
I deleted the _rowy_ collection in my Firestore earlier when I was fixing the FlutterFlow bug, and am now encountering this error.

**Comment by harinij** [comment, marked as answer]
@organicnz this issue should be resolved in the recent releases. We also have a delete project option for use instead of having to manually delete things. In case you are still facing any issues with Rowy Run setup let us know, if not closing out this for now.
